app/_src/gateway/kong-manager/workspaces.md

```diff
@@ -75,45 +75,34 @@ navigate to the new Workspace's dashboard.
 
 ## Delete a workspace
 
-### Wipe workspace data
 To delete a workspace, *all data* must first be deleted from the workspace.
 Choose one of the following methods.
 
 {% navtabs %}
 {% navtab Kong Manager %}
 Using Kong Manager, complete the following:
 
-1.  Manually delete all files via **Dev Portal** > **Editor**. You cannot delete folders at this time, but deleting
-all files from a folder will remove the folder.
-1. Turn off the Dev Portal. Go to Dev Portal **Settings** > **Advanced** > **Turn Off**.
-1. Remove all roles from the workspace:
-     1. Go to the **Teams** tab.
-     1. Navigate to the **Roles** tab.
-     1. Click **View** on the workspace you want to delete.
-     1. Go to each role entry and click **Edit**.
-     1. In the entry detail page, click **Delete Role**. A confirmation modal will appear. Click **Delete Role** again.
+1. Go to **Workspaces** and select the workspace you want to delete.
+
+1. Click **Settings** and then **Delete**.
+
+1. In the **Delete Workspace** dialog, enter the name of the workspace, select **Confirm: delete all associated resources**, and then click **Delete**. 
+
+This will automatically delete all entities (teams and roles) associated with the workspace as well as the workspace itself.
 
 {% endnavtab %}
 {% navtab Admin API %}
 
-1. Delete all Dev Portal files associated with the workspace:
+1. Delete the workspace and [all entities associated with the workspace](/gateway/{{page.kong_version}}/admin-api/workspaces/reference/#delete-a-workspace):
```

Comment:
```suggestion
Delete the workspace and [all entities associated with the workspace](/gateway/{{page.kong_version}}/admin-api/workspaces/reference/#delete-a-workspace):
```
Don't need the number since there's only one step.